app/src/main/java/org/mozilla/focus/search/SearchEngine.java

```diff
@@ -61,7 +61,7 @@ public String buildSearchUrl(final String searchTerm) {
 
         final String template = Uri.decode(searchUri.toString());
         final String urlWithSubstitutions = paramSubstitution(template, Uri.encode(searchTerm));
-        return UrlUtils.normalize(urlWithSubstitutions); // User-entered search engines may need normalization.
+        return UrlUtils.INSTANCE.normalize(urlWithSubstitutions); // User-entered search engines may need normalization.
```

Comment:
fwiw, it'd be helpful to mention in the comment summary that this was an auto conversion with no modifications.